pkg/controller/ec2/vpcpeeringconnection/setup.go

```diff
@@ -48,6 +52,7 @@ func SetupVPCPeeringConnection(mgr ctrl.Manager, l logging.Logger, rl workqueue.
 			managed.WithExternalConnecter(&connector{kube: mgr.GetClient(), opts: opts}),
 			managed.WithCreationGracePeriod(3*time.Minute),
 			managed.WithLogger(l.WithValues("controller", name)),
+			managed.WithInitializers(managed.NewDefaultProviderConfig(mgr.GetClient()), managed.NewNameAsExternalName(mgr.GetClient()), &tagger{kube: mgr.GetClient()}),
```

Comment:
```suggestion
			managed.WithInitializers(managed.NewNameAsExternalName(mgr.GetClient()), &tagger{kube: mgr.GetClient()}),
```

Provider config default is deprecated in favor of CRD defaulting.